### ISSUE TITLE
训练时候的steps和具体current_examplede 数量有对应关系吗？

### ISSUE THREAD

**Issue by lsy641** [PaddlePaddle/Paddle#19732]
这是一次训练刚开始的verbose：
[INFO] 2019-09-10 03:26:25,007 [ run_chid.py:  280]:	train pyreader queue size: 27, learning rate: 0.000065
[INFO] 2019-09-10 03:26:25,010 [ run_chid.py:  292]:	epoch: 0, progress: 3911/571381, step: 2410, ave loss: 3.670118, ave acc: 0.817708, speed: 0.816043 steps/s
[INFO] 2019-09-10 03:26:33,834 [ run_chid.py:  280]:	train pyreader queue size: 50, learning rate: 0.000065
[INFO] 2019-09-10 03:26:33,835 [ run_chid.py:  292]:	epoch: 0, progress: 7296/571381, step: 2420, ave loss: 3.362115, ave acc: 0.796875, speed: 1.133226 steps/s
[INFO] 2019-09-10 03:26:42,704 [ run_chid.py:  280]:	train pyreader queue size: 50, learning rate: 0.000065
[INFO] 2019-09-10 03:26:42,704 [ run_chid.py:  292]:	epoch: 0, progress: 9216/571381, step: 2430, ave loss: 2.915945, ave acc: 0.822917, speed: 1.127665 steps/s
[INFO] 2019-09-10 03:26:51,692 [ run_chid.py:  280]:	train pyreader queue size: 50, learning rate: 0.000065

第一次为什么一次就处理了3911个examples啊？看不懂。我是设置steps=2400开始训练的，10个steps过后就是3911个examples，也不是batch_sizede 整数倍，虽然后面的每十个step是batch_size的十倍，为什么第一个不是呢?

**Comment by NHZlX** [comment]
batch size是多少呢
